NOTE:
you need to update checksum when you modify docuwiki.php

--- docuwiki/conf/dokuwiki.php~	2007-06-26 21:27:15.000000000 +0300
+++ docuwiki/conf/dokuwiki.php	2007-08-19 19:39:43.468686519 +0300
@@ -18,7 +18,7 @@
 $conf['lang']        = 'en';              //your language
 $conf['basedir']     = '';                //absolute dir from serveroot - blank for autodetection
 $conf['baseurl']     = '';                //URL to server including protocol - blank for autodetect
-$conf['savedir']     = './data';          //where to store all the files
+$conf['savedir']     = '/var/lib/dokuwiki';          //where to store all the files
 $conf['allowdebug']  = 0;                 //allow debug output, enable if needed 0|1
 
 /* Display Options */
--- dokuwiki-2008-05-05/install.php~	2008-05-05 21:33:44.000000000 +0300
+++ dokuwiki-2008-05-05/install.php	2008-05-05 21:33:49.877839346 +0300
@@ -44,6 +44,8 @@
     '2007-05-24'   => 'd80f2740c84c4a6a791fd3c7a353536f',
     '2007-06-26'   => 'b3ca19c7a654823144119980be73cd77',
     '2008-05-04'   => '1e5c42eac3219d9e21927c39e3240aad',
+	# md5sum -b conf/dokuwiki.php
+    '2008-05-04-pld'   => 'b3f453c8e68e7a06c312c230f35bcdd3',
 );
 
 
